docs/content/documentation/content/taxonomies.md

```diff
@@ -3,7 +3,62 @@ title = "Taxonomies"
 weight = 90
 +++
 
-Zola has built-in support for taxonomies.
+Zola has built-in support for taxonomies. Taxonomies are a way for users to group content according to user-defined catagories.
+
+## Definitions
+
+- Taxonomy: A catagorisation that can be used to group content 
+- Term: A specific group within a taxonomy 
+- Value: A peice of content that can be associated with a term
+
+## Example: a movie website
+
+Imagine that you want to make a website to display information about various movies. In that case you could use the following taxonomies:
+
+- Director
+- Genres
+- Awards
+- Release year
+
+Then at build time Zola can create pages for each taxonomy listing all of the known terms, and pages for each term in a taxonomy, listing all of the peices of content assoicated with that term. 
+
+Imagine again we have the following movies: 
+```yaml
+- Shape of water:                   <--- Value
+  - Director:                         <--- Taxnomomy
+    - Guillermo Del Toro                 <--- Term
+  - Genres:                            <--- Taxnomomy
+    - Thriller                           <--- Term
+    - Drama                              <--- Term
+  - Awards:                           <--- Taxnomomy
+    - Golden globe                         <--- Term
+    - Academy award                        <--- Term
+    - BAFTA                                <--- Term
+  - Release year:                      <--- Taxnomomy
+    - 2017                                <--- Term
+    
+- The Room:                         <--- Value
+  - Director:                           <--- Taxnomomy
+    - Tommy Wiseau                         <--- Term
+  - Genres:                              <--- Taxnomomy
+    - Romance                              <--- Term
+    - Drama                                <--- Term
+  - Release Year:                       <--- Taxnomomy
+    - 2003                                 <--- Term
+
+- Bright:                           <--- Value
+  - Director:                           <--- Taxnomomy
+    - David Ayer                           <--- Term
+  - Genres:                              <--- Taxnomomy
+    - Fantasy                              <--- Term
+    - Action                               <--- Term
+  - Awards:                             <--- Taxnomomy
+    - California on Location Awards        <--- Term
+  - Release Year:                       <--- Taxnomomy
+    - 2017                                 <--- Term
+```
```

Comment:
let's not use YAML in the docs

Reply:
okay, do you have a preference for a different format? just a plain nested list perhaps then?